distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -175,28 +183,86 @@ public void testCswRegistryCreateDup() throws Exception {
                 .post(CSW_PATH.getUrl());
     }
 
-    @Ignore
     @Test
     public void testCswRegistryStoreCreate() throws Exception {
-        //Stub test, waiting for additional features/service before implementing
-        //Use federation admin to create a registry entry
-        //on a 'remote' (aka loopback) catalog store
+        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
```

Comment:
Oh, that is because you're trying to directly grab the service and add it in the `createRegistryStoreEntry()` method. Is there a way that you can call that externally via the REST client? Usually we don't directly interact with the services in the integration tests, as these tests are meant to demonstrate how these things would work with an external integrator, or we set things up (like adding entries) before we run the tests. At that point it is acceptable to directly poke the container.